client/cli/tx.go

```diff
@@ -54,3 +68,114 @@ func NewCmdExecute() *cobra.Command {
 
 	return cmd
 }
+
+// NewCmdSubmitUpgrade schedule an on-chain software upgrade.
+// This command has been adapted from the SDK's `NewCmdSubmitUpgradeProposal`.
+// https://github.com/cosmos/cosmos-sdk/blob/x/upgrade/v0.1.4/x/upgrade/client/cli/tx.go#L47
+func NewCmdSubmitUpgrade() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "software-upgrade <name> [--upgrade-height <height>] [--upgrade-info <info>] [flags]",
+		Args:  cobra.ExactArgs(1),
+		Short: "Schedule a software upgrade on chain",
+		Long: "Schedule a software upgrade on chain.\n" +
+			"Please specify a unique name and height for the upgrade to take effect.\n" +
+			"You may include info to reference a binary download link, in a format compatible with: https://docs.cosmos.network/main/build/tooling/cosmovisor",
```

Comment:
Also think this could you a bit of a cleanup too!

```suggestion
		Short: "Helper for scheduling a software upgrade",
		Long: "Helper for scheduling a software upgrade.\n\n" +
			"You can additionally include upgrade info via a flag to reference pre-built binaries, documentation, etc.\n" +
			"https://docs.cosmos.network/main/build/tooling/cosmovisor",
```